core/search_test.go

```diff
@@ -3,69 +3,109 @@ package core
 import (
 	"testing"
 
-	"github.com/qri-io/qri/repo"
+	// "github.com/qri-io/qri/repo"
```

Comment:
let's delete this. Apologies for introducing the bad habit of commenting out imports, I'm hoping we can all avoid it in the future 😃